container/gke-implementation-guide.md

```diff
@@ -129,7 +129,8 @@ Various command-line utilities are required for this demo. The utilities can eit
    NAME         LOCATION    MASTER_VERSION    MASTER_IP     MACHINE_TYPE  NODE_VERSION      NUM_NODES  STATUS
    gke-cluster  us-east1-b  1.17.14-gke.1600  12.345.15.12  e2-medium     1.17.14-gke.1600  3          RUNNING
    ```
-
+ - Once the cluster is created, before accessing the cluster, run the command ```gcloud container clusters get-credentials <cluster name>``` , which will update the ```kubeconfig``` file with credentials and endpoint information to point ```kubectl``` to the newly created cluster.
+ 
```

Comment:
```suggestion
 - Once the cluster has been created and before attempting to access the cluster, run the command `gcloud container clusters get-credentials <cluster name>`. This will update the `kubeconfig` file with cluster credentials and configuration which will switch the `kubectl` context to point to the newly created cluster.
 
```

Reply:
Hey @MPH13 , I don't have write permissions to your repo, so you'll need to take on these suggestions :)